Automattic/newspack-plugin
# Update gray colors using latest WordPress base-style

### All Submissions:






### Changes proposed in this Pull Request:

This PR updates @wordpress/base-styles to 3.4.4 which means variables like `$dark-gray-900` are gone... so I had to replace all the color variables with the new ones.

### How to test the changes in this Pull Request:

1. Switch to this branch
2. `npm i && npm rebuild node-sass && npm run build`
3. check all the various wizards (and let me know if you notice any issue!)

### Other information:
